Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs

```diff
@@ -91,23 +106,13 @@ private void UpdateSecondSolverHandler()
         {
             if (secondSolverHandler != null)
             {
-                if (secondTransformOverride != null)
-                {
-                    secondSolverHandler.TransformTarget = secondTransformOverride;
-                }
-                else
+                secondSolverHandler.TrackedTargetType = secondTrackedObjectType;
+
+                if (secondTrackedObjectType == TrackedObjectType.CustomOverride && secondTransformOverride != null)
                 {
-                    secondSolverHandler.TrackedObjectToReference = trackedObjectForSecondTransform;
+                    secondSolverHandler.SetTransformOverride(secondTransformOverride);
                 }
             }
         }
-
-        /// <summary>
-        /// This should only be called from the SolverInBetweenEditor to cause the secondary SolverHandler to reattach this solver.
-        /// </summary>
-        public void AttachSecondTransformToNewTrackedObject()
```

Comment:
>AttachSecondTransformToNewTrackedObject [](start = 20, length = 39)

Please add migration notes on how to accomplish this now (which is just updating the property itself!)